Detectors/TPC/calibration/include/TPCCalibration/CalibdEdx.h

```diff
@@ -63,18 +61,18 @@ class CalibdEdx
   // and the HistAxis enum
   using AxesType = std::tuple<
     FloatAxis, // dEdx
-    FloatAxis, // Z
     FloatAxis, // Tgl
-    // FloatAxis, // Snp
+    FloatAxis, // Snp
+    // FloatAxis, // Z
     IntAxis, // sector
     IntAxis, // stackType
     IntAxis  // Charge
     >;
 
   using Hist = boost::histogram::histogram<AxesType>;
-  using FitCuts = std::array<int, 3>;
+  using FitCuts = std::array<float, 3>;
```

Comment:
Why did you change this to float?

Reply:
For stupid reasons, never mind. Its back an int.